frontend/src/components/landing/Plans.tsx

```diff
@@ -30,64 +31,160 @@ export const Plans = (props: Props) => {
     label: "newlanding-plans-button",
   });
 
+  /** List of premium features **/
+  const premiumFeatures = (
+    <ul className={styles.features}>
+      <li>{l10n.getString("landing-pricing-premium-feature-1-2")}</li>
+      <li>{l10n.getString("landing-pricing-premium-feature-2")}</li>
+      <li>{l10n.getString("landing-pricing-premium-feature-3-2")}</li>
+      <li>{l10n.getString("landing-pricing-premium-feature-4")}</li>
+      <li>{l10n.getString("landing-pricing-premium-feature-5")}</li>
+    </ul>
+  );
+
+  /** List of free features **/
+  const freeFeatures = (
+    <ul className={styles.features}>
+      <li>{l10n.getString("landing-pricing-free-feature-1-2")}</li>
+      <li>{l10n.getString("landing-pricing-free-feature-2")}</li>
+    </ul>
+  );
+
+  /** If premium is not available, show waitlist panel **/
+  const unavailablePremiumPanel = (
+    <a
+      href="/premium/waitlist"
```

Comment:
@Vinnl Is there a specific format to link to different pages? When clicking this while running `npm run dev` I get this error: 
<img width="882" alt="image" src="https://user-images.githubusercontent.com/13066134/175349151-54ec4f16-da48-4705-bea6-cf16772dc1e3.png">

Reply:
There is, although the error is unrelated I think - it looks like you're missing the `cldr-localenames-modern` package, so an `npm install` should fix that.

That said, when linking to other pages on the same website, you'll need to [use Next.js's `<Link>` component](https://nextjs.org/docs/routing/introduction#linking-between-pages) to ensure page navigations without a full refresh.